src/sourcemap/CodeCoverage.zig

```diff
@@ -388,8 +446,12 @@ pub const ByteRangeMapping = struct {
             line_count = @truncate(line_starts.len);
             executable_lines = try Bitset.initEmpty(allocator, line_count);
             lines_which_have_executed = try Bitset.initEmpty(allocator, line_count);
-            lines_hits = try LinesHits.initCapacity(allocator, line_count);
-            errdefer lines_hits.deinitWithAllocator(allocator);
+            line_hits = try LinesHits.initCapacity(allocator, line_count);
+            line_hits.len = line_count;
+            const line_hits_slice = line_hits.slice();
+            @memset(line_hits_slice, 0);
```

Comment:
@exoego note how we now zero-initialize the `line_hits` array to ensure that we can access values which have never been written to without worrying about undefined behavior.